.github/workflows/ghcrCleanup.yml

```diff
@@ -1,26 +1,48 @@
 name: Delete old container images
 
-
 on:
   schedule:
     - cron: "0 0 * * *"
 
-permissions:
-  packages: write
-
 jobs:
   clean_ghcr:
-    name: Delete old unused container images
+    name: Delete untagged container images
     runs-on: ubuntu-latest
     steps:
-      - name: Delete all containers older than a month
-        uses: snok/container-retention-policy@v2
-        with:
-          image-names: backend, cypress, database, frontend, frontend-lighthouse, nginx
-          cut-off: One month ago UTC
-          account-type: org
-          org-name: CDCgov
-          keep-at-least: 1
-          skip-tags: main
-          token: ${{ secrets.GITHUB_TOKEN }}
-          dry-run: true
+      # backend, cypress, database, frontend, frontend-lighthouse, nginx
+      - uses: actions/delete-package-versions@v4
+        with: 
+          package-name: 'backend'
+          package-type: 'container'
+          min-versions-to-keep: 10
```

Comment:
(for my own knowledge 😅) if we are only deleting untagged versions is there a reason we need to keep a minimum of 10 versions of them? 🤔

Reply:
I don't have a good reason except that I want to see the arguments at work. :smiley:

Reply:
that sounds good to me! I would also like to see that too! 😹